### PR TITLE
Refactor/remove any types from EA

### PR DESCRIPTION
**Pull request by bogos**
## Description

Remove any types from EA

## Steps to Test

yarn lint 

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [ ] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

File: packages/core/test-helpers/src/helpers.ts [hunk 1]
```diff
@@ -3,7 +3,9 @@ import { AdapterRequest, Execute } from '@chainlink/ea-bootstrap'
 
 const mockContext = {}
 
-export function assertError(statusCode: any, data: any, expectedJobId: any) {
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function assertError(statusCode: any, data: any, expectedJobId: any): void {
```
Comment (boxhock) [inline packages/core/test-helpers/src/helpers.ts:8]: Are these really necessary? Seems like we should be able to get types for a few of these, right?

File: packages/sources/1forge/src/adapter.ts [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const makeExecute: ExecuteFactory<DefaultConfig, endpoints.TInputParamete
   return async (request, context) => execute(request, context, config || makeConfig())
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (boxhock) [inline packages/sources/1forge/src/adapter.ts:39]: Is this one also needed?

File: packages/sources/amberdata/src/adapter.ts [hunk 3]
```diff
@@ -48,6 +48,7 @@ interface Message {
   }
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
 export const makeWSHandler = (defaultConfig?: DefaultConfig): MakeWSHandler<Message | any> =>
```
Comment (boxhock) [inline packages/sources/amberdata/src/adapter.ts:52]: Would it be better to do Message | UnknownWSMessage instead, so we avoid disabling the any check?

File: packages/sources/cfbenchmarks/src/adapter.ts [hunk 4]
```diff
@@ -38,6 +38,7 @@ interface Message {
   time: number
 }
 
+/* eslint-disable @typescript-eslint/no-explicit-any */
 export const makeWSHandler = (config?: Config): MakeWSHandler<Message | any> =>
```
Comment (boxhock) [inline packages/sources/cfbenchmarks/src/adapter.ts:42]: Same as above

**Review by boxhock — CHANGES_REQUESTED** [review]
Only had a quick look for now. Mostly had questions around accepting `any` in WS EAs. Would it not be better to use the unknown WS message type instead of explicitly disabling the `any` check there?

**Comment by alejoberardino** [comment]
Closing this, as the new framework supersedes this work

**Pull request closed without merging**